keystone-init/README.md

```diff
@@ -174,6 +174,63 @@ containers:
 {{ include "keystone_env" .Values.my_pod.auth | indent(6) }}
 ```
 
+Using external OpenStack Keystone
```

Comment:
I think that this documentation, presented in this form, is better to be put to ```monasca-docker/README.md``` instead of here. We have a bunch of examples how to connect to keystone and each of them is not about ```keystone-init```